src/storybook.d.ts

```diff
@@ -3,6 +3,11 @@ export interface Options {
   hideArgRef?: boolean;
   /** sets the custom type reference in the Custom Elements Manifest */
   typeRef?: string;
+  /**
+   * doesn't render attributes when their value is equal to the default value
+   * of that attribute
+   */
+  hideDefaultAttributeValues?: boolean;
```

Comment:
Do you think we need a configuration for this or should this be the default behavior? I kind of feel like it should be the default behavior.

Reply:
Agree, I think it makes more sense as default behaviour, with the option to enable it the other way around in case somebody prefers 100% clarity what the attribute values are in their documentation. I can convert the setting into `renderDefaultAttributeValue`? Should there maybe be a sourceCode/template sub-setting which includes the options `showDefaultAttributeValues` and `hideScriptTag`? So that it's clear, it's not about the default attribute behaviour in the controls panels?

Reply:
Yeah, I like `renderDefaultAttributeValue`. I also like the idea of this being configurable at the story level as well in the `getWcStorybookHelpers` function.